### PR TITLE
Added language for GDPR and CCPA PM requests

### PR DESCRIPTION
**Pull request by Nevazhnovu**

### CODE REVIEW

**Comment by andresilveirah** [comment]
@Nevazhnovu can you add tests for it?

File: Example/ConsentViewController_ExampleTests/SPClientCoordinator/SourcePointClient/SourcePointClientSpec.swift [hunk 1]
```diff
@@ -78,6 +79,18 @@ class SourcePointClientSpec: QuickSpec {
...
+                let pmUrl = Constants.Urls.GDPR_PM_URL.appendQueryItems([
+                    "message_id": "111",
+                    "pmTab": "",
+                    "consentUUID": "xxxxxx-xxxx-xxxx-8xxxxad-xxxxxxx",
+                    "idfaStatus": "accepted",
+                    "site_id": "111",
+                    "consentLanguage": SPMessageLanguage.Spanish.rawValue
+                ])
+                expect(pmUrl?.absoluteString) == "https://cdn.privacy-mgmt.com/privacy-manager/index.html?consentLanguage=ES&consentUUID=xxxxxx-xxxx-xxxx-8xxxxad-xxxxxxx&idfaStatus=accepted&message_id=111&pmTab=&site_id=111"
+            }
```
Comment (andresilveirah) [inline Example/ConsentViewController_ExampleTests/SPClientCoordinator/SourcePointClient/SourcePointClientSpec.swift:93]: @Nevazhnovu I believe the test above doesn't fully test the feature. 

What we're aiming for is:
1. GIVEN the user sets `.messageLanguage` to `.Spanish` on `SPConsentManager` object
2. AND the user calls `loadGDPRPrivacyManager` (or the CCPA counter part)
3. THEN the loaded URL includes the query param `consentLanguage=ES`

File: Example/ConsentViewController_ExampleTests/SPConsentManagerSpec.swift [hunk 2]
```diff
@@ -51,5 +51,26 @@ class SPConsentManagerSpec: QuickSpec {
...
+                    expect(pmUrl?.absoluteString)=="https://cdn.privacy-mgmt.com/privacy-manager/index.html?consentLanguage=ES&consentUUID&idfaStatus=accepted&message_id=1&pmTab=&site_id=1"
+                }
+            }
+            describe("ccpa") {
+                it("build URL with the right parameters") {
+                    var manager = SPConsentManager(accountId: 1, propertyId: 1, propertyName: coordinator.propertyName, campaigns: coordinator.campaigns, delegate: nil)
+                    manager.messageLanguage = .Spanish
+                    var pmUrl = manager.buildCCPAPmUrl(usedId: "1")
+
+                    expect(pmUrl?.absoluteString)=="https://cdn.privacy-mgmt.com/ccpa_pm/index.html?consentLanguage=ES&consentUUID&idfaStatus=accepted&message_id=1&pmTab=&site_id=1"
```
Comment (Nevazhnovu) [inline Example/ConsentViewController_ExampleTests/SPConsentManagerSpec.swift:71]: @andresilveirah my local SPConsentManager sets `idfaStatus=accepted` but GitHub says `unknown`. 
I could hardcode but I will make a public-getter property. 
Either way, it is a lame approach but no idea how to do it better with existing `SPConsent` architecture.

Comment (andresilveirah) [reply Example/ConsentViewController_ExampleTests/SPConsentManagerSpec.swift]: I think your approach is valid. [There are ways to reset the IDFA status before the test](https://github.com/SourcePointUSA/ios-cmp-app/blob/9fc5b7ae5064fe168ab7dc0177c7f9d0924c1569/Example/SPGDPRExampleAppUITests/ExampleApp.swift#L27), but it's not available before iOS 15.

**Review by andresilveirah — APPROVED** [review]

**Pull request merged**